server/server.js

```diff
@@ -7,6 +7,44 @@ const app = express()
 app.use(bodyParser.urlencoded({ extended: false }))
 app.use(bodyParser.json())
 
+const nodemailer = require('nodemailer')
```

Comment:
type `import nodemailer from 'nodemailer'` and put below the `import bodyParser ... ` -line